README.md

```diff
@@ -36,6 +36,7 @@ To get a local copy up and running follow these simple example steps.
 
 ## Authors
 - Website: [@My Portfolio](https://lionrouge1.github.io/Portfolio/)
+- My portfolio: [@Portfolio](https://lionrouge1.github.io/Portfolio/)
 
 👤 **Author1**
 
```

Comment:
[OPTIONAL]
- You have added the link to your portfolio to Readme in a good way and it works. Let's make the Readme file more professional by adding a section of demo link to make the visitor able to find the demo link of your app easily.
```md
### Demo
[Demo link](http...)
```

Reply:
Thank you for the review.